packages/nuejs.org/@lib/video.css

```diff
@@ -48,7 +47,7 @@
   height: 4.2em;
   width: 4.2em;
   left: 50%;
-  top: 45%;
+  top: calc(50% - 2.1em);
```

Comment:
Todo: Use variable e.g. `--size` for width, height and top: `calc(50% - var(--size) / 2)`